mkdocs.yml

```diff
@@ -62,29 +66,30 @@ nav:
   - quickstart.md
   - 'User Guide':
     - user-guide/devguide.md
-    - user-guide/architecture.md
-    - user-guide/message-flows.md
     - user-guide/partner.md
-    - user-guide/caching.md
     - user-guide/upgrade.md
     - user-guide/documentation.md
     - user-guide/observability.md
-    - user-guide/ci-setup.md
     - 'Basic Station configuration':
       - user-guide/station-device-provisioning.md
       - user-guide/station-authentication-modes.md
       - user-guide/station-module-configuration.md
     - 'Region specifics':
-      - user-guide/as923.md
+      - user-guide/as923.md  
+    - Testing:
+      - testing/e2e_tests.md
+      - testing/integration_tests.md
+  - 'Reference':
```

Comment:
Should this section be under 'User Guide', since files under it are located in that folder? The indentation makes it look like it's not under User Guide.

Reply:
you're right, I've moved the files under user-guide for clarity